src/ess/livedata/dashboard/reduction.py

```diff
@@ -39,7 +49,9 @@ def create_main_content(self) -> pn.viewable.Viewable:
 
 
 def get_arg_parser() -> argparse.ArgumentParser:
-    return Service.setup_arg_parser(description='ESSlivedata Dashboard')
+    return Service.setup_arg_parser(
+        description='ESSlivedata Dashboard', dashboard_auth=True
+    )
```

Comment:
Since auth is only for the dashboard, it might make more sense to add the options here before returning, instead of adding a flag to the to the generic `Service`, which is used by the backend workers as well?